components/browser/menu/src/test/java/mozilla/components/browser/menu/WebExtensionBrowserMenuBuilderTest.kt

```diff
@@ -4,28 +4,54 @@
 
 package mozilla.components.browser.menu
 
+import android.view.LayoutInflater
 import android.view.View
 import android.widget.ImageButton
 import androidx.recyclerview.widget.RecyclerView
 import androidx.test.ext.junit.runners.AndroidJUnit4
+import mozilla.components.browser.menu.item.BrowserMenuImageText
 import mozilla.components.browser.menu.item.WebExtensionBrowserMenuItem
+import mozilla.components.browser.menu.item.WebExtensionBrowserNestedMenuItem
 import mozilla.components.browser.state.state.BrowserState
 import mozilla.components.browser.state.state.WebExtensionState
 import mozilla.components.browser.state.store.BrowserStore
 import mozilla.components.concept.engine.webextension.WebExtensionBrowserAction
 import mozilla.components.support.test.mock
 import mozilla.components.support.test.robolectric.testContext
 import org.junit.Assert.assertEquals
+import org.junit.Assert.assertFalse
 import org.junit.Assert.assertNotNull
+import org.junit.Assert.assertTrue
 import org.junit.Test
 import org.junit.runner.RunWith
-import org.mockito.Mockito
 
 @RunWith(AndroidJUnit4::class)
 class WebExtensionBrowserMenuBuilderTest {
 
     @Test
-    fun `web extension is added at the start when appendExtensionActionAtStart is true`() {
+    fun `WHEN there are no web extensions installed THEN add-ons menu item invokes onAddonsManagerTapped`() {
```

Comment:
nit: when there are no web extension actions...

(We could have add-ons installed that don't have browser actions :)